### ISSUE TITLE
Indentation of var declarations

### ISSUE THREAD

**Issue by avaragado** [douglascrockford/JSLint#53]
I like to indent var declarations for readability. Here's an example (with contrived data for the purposes of this issue):

``` javascript
var a, b,
  c = 1,
  x = 2 +
    3,
  foo = 234 +
    4;
```

In particular, if an initialised variable needs to wrap I indent further. However, JSLint (2011-06-22) reports errors in these cases, telling me in this example that I've indented the '3' and '4' two characters too far.

In the common use case, which the above is not, reducing the intentation of the wrapped code or bracketing the expression would make it less readable. Unwrapping would also be a poor solution.

Is JSLint behaving correctly by reporting these as errors? If so, how should I be writing these types of declarations? Thanks.
